test/compiler/long_file/run.t

```diff
@@ -0,0 +1,14 @@
+Degenerate performance case for the fcc compiler
+  $ export FCC_TEST=true
+
+We now compile the challenging file:
+  $ fcc --root . ./test_2.v
```

Comment:
I may be nicer to regenerate the file here where you can run arbitrary shell rather than put an unreviewable 30k lines file in the git commit.

Reply:
Good idea, done thanks!